README.md

```diff
@@ -107,7 +107,9 @@ The following apps are supported by govuk-docker to some extent.
    - ⚠  collections
       * You will need to [populate the Content Store database](#mongodb) or run the live stack in order for it to work locally.
       * To view topic pages locally you still need to use the live stack as they rely on Elasticsearch data which we are yet to be able to import.
-   - ✅ collections-publisher
+   - ⚠ collections-publisher
+      * 'Checking for broken links' in step by step requires link-checker-api to be running in another process (see link-checker-api below).
```

Comment:
I think there's a potential for confusion here, since link-checker-api will be running as part of docker-compose dependency management. Can we clarify this to indicate the problem is with link-checker-api not working (vs. docker-compose not starting the process)?

Reply:
Have rewritten - please check.